op-acceptance-tests/tests/base_multi/advance_multiple.go

```diff
@@ -0,0 +1,45 @@
+package base_multi
+
+import (
+	"os"
+	"testing"
+	"time"
+
+	"github.com/ethereum-optimism/optimism/op-devstack/devtest"
+	"github.com/ethereum-optimism/optimism/op-devstack/presets"
+	"github.com/stretchr/testify/require"
+)
+
+// TestCLAdvanceMultiple verifies two L2 chains advance when using separate CLs
+func TestCLAdvanceMultiple(gt *testing.T) {
```

Comment:
Should these new AT's be placed in the `base` gate?
Related to that, would you be happy to introduce them via the new [flake-shake gate](https://github.com/ethereum-optimism/optimism/blob/develop/op-acceptance-tests/acceptance-tests.yaml#L8-L37)?